app/javascript/src/apis/contact.js

```diff
@@ -0,0 +1,39 @@
+const fetch = async () => {
+  const userExample1 = {
+    name: "Ronald Richards",
+    role: "Owner",
+    email: "richards@example.com",
+    profile_pic: "https://i.pravatar.cc/300",
+    created_at: new Date().toString(),
+  };
+  const userExample2 = {
+    name: "Jacob Jones",
+    role: "Owner",
+    email: "jones@example.com",
+    profile_pic: "",
+    created_at: new Date().toString(),
+  };
+  const mock = [];
+  let count = 0;
+  for (let i = 0; i < 50; i++) {
+    userExample1["id"] = count;
+    userExample2["id"] = count + 1;
+    count = count + 2;
+    mock.push(JSON.parse(JSON.stringify(userExample1)));
+    mock.push(JSON.parse(JSON.stringify(userExample2)));
+  }
+  return {
+    data: {
+      contacts: mock,
+    },
+  };
+};
+const create = async () => {};
+const destroy = async () => {};
```

Comment:
You should be resolving Promises from these functions. Otherwise it doesn't make sense to put them in async block and await them wherever they're invoked.

Reply:
Yedhin just to ensure that I fully understand. Is it okay to do this way?
<img width="688" alt="Screenshot 2022-09-23 at 7 37 03 PM" src="https://user-images.githubusercontent.com/56035039/191979316-2b6d8b59-aa70-4a43-90b2-d64526d9b69d.png">